docs/user/intro/mkdocs.rst

```diff
@@ -26,37 +19,58 @@ specifying a python toolchain on Ubuntu, defining the location of the installati
     build:
       os: ubuntu-24.04
       tools:
-        python: "3"
-
-    python:
-      install:
-        - requirements: requirements.txt
+        python: 3
+      jobs:
+        pre_install:
+          - pip install mkdocs
+          - pip install mkdocs-material
```

Comment:
```suggestion
      jobs:
        post_install:
          - pip install mkdocs-material
```

If we are using `mkdocs` config key, we only need to install `mkdocs-material` after the `install` job was executed.